WordPress/src/test/java/org/wordpress/android/ui/mysite/MySiteViewModelTest.kt

```diff
@@ -848,9 +848,10 @@ class MySiteViewModelTest : BaseUnitTest() {
     }
 
     @Test
-    fun `site info card url click opens site`() = test {
+    fun `site info card url click opens site and requests next step of View Site task`() = test {
         invokeSiteInfoCardAction(SiteInfoHeaderCardAction.URL_CLICK)
 
+        verify(quickStartRepository).completeTask(QuickStartTask.VIEW_SITE)
         assertThat(navigationActions).containsOnly(SiteNavigationAction.OpenSite(site))
```

Comment:
Nitpick: Considering that there's no next step for `View Site` task, how about we change `and requests next step of View Site task` to `and completes View Site task`?